src/rust/src/x509.rs

```diff
@@ -291,6 +325,55 @@ fn parse_name(py: pyo3::Python<'_>, name: Name<'_>) -> Result<pyo3::PyObject, Py
     Ok(py_name)
 }
 
+fn ipv4_netmask(data: &[u8]) -> Result<u32, PyAsn1Error> {
```

Comment:
Make this take a `u32` and do the conversion from `&[u8]` in the caller so it's obviously correct